drivers/media/i2c/imx258.c

```diff
@@ -110,7 +110,7 @@ struct imx258_mode {
 	struct imx258_reg_list reg_list;
```

Comment:
There's a space missing from `(Y_ADD_END – Y_ADD_STA+ 1)` in the commit message.